docs/config.md

```diff
@@ -86,6 +86,12 @@ can use the following command:
 eden controller -m adam:// edge-node update --config timer.config.interval=5
 ```
 
+To set options for virtualized environment please use several options before run of `eden setup`:
```

Comment:
Could you please elaborate why one should change it. I believe it's for the eve itself?

Reply:
For example we must adjust resources of EVE itself if we plan to start VM inside with additional resources (more CPUs/ram/disk).